src/components/checkbox/checkbox.scss

```diff
@@ -6,16 +6,23 @@ $checkbox-border-width: 2px !default;
 $checkbox-text-margin: 10px !default;
 $checkbox-top-left: 12px !default;
 
+.md-inline-form {
+  md-checkbox {
+    margin: 19px 0 18px;
```

Comment:
These same values are used in switch, checkbox, and radio-group. Could they be scss variables?